modules/oe_webtools_geocoding/config/schema/oe_webtools_geocoding.schema.yml

```diff
@@ -1,3 +1,9 @@
 geocoder_provider.configuration.webtools_geocoding:
   type: geocoder_provider_configuration
   label: 'Webtools Geocoding configuration'
+  mapping:
+    referer:
+      type: string
```

Comment:
The core's `uri` type is more appropriate here.

Reply:
Fine for me, the reason I used a string is because it is consistent with `geocoder_provider.configuration.nominatim` which is defined in `geocoder.schema.yml`.

I agree that a `uri` is more appropriate.